portfolio/cspell.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
`cspell` is a really helpful tool that will catch any spelling mistakes in your code. It has saved me a thousand times before, so I pretty much always loop it in to any projects I work on. If you are using VSCode, you can install the `cspell` extension which will also work alongside you as you code and give you the option to fix misspellings as well as add new words to this file to be ignored by `cspell`